src/common/checksum.c

```diff
@@ -236,7 +264,8 @@ daos_csum_type2algo(enum DAOS_CSUM_TYPE type)
 
 int
 daos_csummer_init(struct daos_csummer **obj, struct csum_ft *ft,
-		  size_t chunk_bytes, bool srv_verify)
+		  size_t chunk_bytes, bool srv_verify, bool dedup,
+		  bool dedup_verify, size_t dedup_bytes)
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?